frontend/src/app/cookie-law-banner/cookie-law-banner.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { CookieService } from 'ngx-cookie';
+
+@Component({
+  selector: 'volontulo-cookie-law-banner',
+  templateUrl: './cookie-law-banner.component.html',
+  styleUrls: ['./cookie-law-banner.component.css']
+})
+export class CookieLawBannerComponent implements OnInit {
+
+  constructor(private _cookieService: CookieService) {
+  }
+
+  ngOnInit() {
+    this.hideCookieBanner(!!this._cookieService.get('cookielaw_accepted'));
+  }
+
+  acceptCookieLaw() {
```

Comment:
I'm not certain, that we have established some standard for `private` members of classes, but it looks that this one could be `private` also.

We need to decide also if we want want to prefix all `private` members using underscore.

Reply:
I set it to not private because it was used in the template. I'll check if it's still available in the template if private and change it if so.

Reply:
Most probably it will not work. Problem occurs only when frontend is build for production, see: https://travis-ci.org/CodeForPoznan/volontulo/builds/277718877#L8103